tests/robustness/failpoints.go

```diff
@@ -85,30 +87,68 @@ var (
 	}}
 )
 
-func triggerFailpoints(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster, config FailpointConfig) {
+func verifyClusterHealth(ctx context.Context, t *testing.T, clus *e2e.EtcdProcessCluster, failpointName string) error {
```

Comment:
Can you move the function below `triggerFailpoints`, they should be ordered top-down.